packages/plugin-ext-vscode/src/node/scanner-vscode.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2015-2018 Red Hat, Inc.
+ * Copyright (C) 2015-2020 Red Hat, Inc.
```

Comment:
Interesting - this project did not yet exist in 2015. Maybe this stems from Red Hat code that predated Theia or a mistake was made. Not this PR's concern, other then updating the most recent year.